mcp/docs/GCP_APP_ENGINE.md

```diff
@@ -111,30 +160,47 @@ attributes specific to only GAE standard
 | `static_files.path` | string | true within static_files context only | | none |
 | `upload_path_regex` | string |  true within static_files context only | | none |
 
-#### Manifest Files
-For GAE deployments with a deployment type of `files`, a `mmcf-manifest.json` manifest file should be included.
-
-By default this file is located in the `<root_dir>/build` directory for the AS, and will contain the keys `artifactDir` and `contents`.
-
-Example:
-```json
-{ 
-  "artifactDir": "exploded-project-app",
-  "contents": [
-    "META-INF/MANIFEST.MF",
-    "WEB-INF/logging.properties",
-    "WEB-INF/classes/logback.xml",
-    "WEB-INF/classes/application.conf",
-    "WEB-INF/lib/annotations-4.1.1.4.jar",
-    "WEB-INF/lib/commons-logging-1.2.jar",
-    "WEB-INF/lib/gson-2.8.6.jar",
-    "WEB-INF/lib/api-common-1.9.0.jar",
-    "WEB-INF/lib/ktor-network-1.3.2.jar",
-    "WEB-INF/lib/ktor-utils-jvm-1.3.2.jar",
-    "WEB-INF/web.xml"
-  ]
-}
+#### Google App Engine Flexible component Configuration  
+An example flexible app engine configuration:  
+```yaml
+project_id: &project_id ae-flex-test
+create_google_project: false
+location_id: "europe-west2"
+components:
+  common:
+    entrypoint: python main.py
+    runtime: python38
+    env: flex
+    env_variables:
+      GCP_PROJECT_ID: *project_id
+  specs:
+    default:
+      version_id: v3
+      deployment:
+        container:
+          image: "europe-west2-docker.pkg.dev/<project_id>/<repository_name>/helloworld:latest" 
+      automatic_scaling: {} #Will use the default automatic_scaling settings
+      resources: 
+        memory_gb: 2
+      readiness_check: 
+        success_threshold: 3
 ```
+***NOTE***: For the container image ensure that the tag or digest is included on the full URI  
+
+#### Deployment Versioning  
+Multiple versions of a service can be deployed at once. 
+* `gcp_ae.yml` will only contain the specifications for one version of each service.  
+* Updating the container version or manifest will not produce a new app engine version, but will update the current deployment
+* If the version key is updated in `gcp_ae.yml` the current deployment will be replaced with the new version
```

Comment:
As there is no traffic managment in this adapter, this will result in the error of trying to destroy a version with non-zero traffic

Reply:
It does? That hasn't been our experience.

Reply:
I meant specifically for the point:
> If the version key is updated in `gcp_ae.yml` the current deployment will be replaced with the new version

Since it replaces the old version, it gives the error:
>  Error when reading or editing AppVersion: googleapi: Error 400: Cannot delete a version with a non-zero traffic allocation. Please update your traffic split to remove the allocation for this version and try again.

Reply:
Should be updated

Reply:
Done